app-modules/service-management/database/migrations/2025_01_03_123821_rename_service_request_resolved_columns_to_closed_on_service_request_types_table.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+<COPYRIGHT>
+
+    Copyright © 2016-2025, Canyon GBS LLC. All rights reserved.
+
+    Aiding App™ is licensed under the Elastic License 2.0. For more details,
+    see <https://github.com/canyongbs/aidingapp/blob/main/LICENSE.>
+
+    Notice:
+
+    - You may not provide the software to third parties as a hosted or managed
+      service, where the service provides users with access to any substantial set of
+      the features or functionality of the software.
+    - You may not move, change, disable, or circumvent the license key functionality
+      in the software, and you may not remove or obscure any functionality in the
+      software that is protected by the license key.
+    - You may not alter, remove, or obscure any licensing, copyright, or other notices
+      of the licensor in the software. Any use of the licensor’s trademarks is subject
+      to applicable law.
+    - Canyon GBS LLC respects the intellectual property rights of others and expects the
+      same in return. Canyon GBS™ and Aiding App™ are registered trademarks of
+      Canyon GBS LLC, and we are committed to enforcing and protecting our trademarks
+      vigorously.
+    - The software solution, including services, infrastructure, and code, is offered as a
+      Software as a Service (SaaS) by Canyon GBS LLC.
+    - Use of this software implies agreement to the license terms and conditions as stated
+      in the Elastic License 2.0.
+
+    For more information or inquiries please visit our website at
+    <https://www.canyongbs.com> or contact us via email at legal@canyongbs.com.
+
+</COPYRIGHT>
+*/
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class () extends Migration {
+    public function up(): void
+    {
+        Schema::table('service_request_types', function (Blueprint $table) {
+            $table->renameColumn('is_managers_service_request_resolved_email_enabled', 'is_managers_service_request_closed_email_enabled');
+            $table->renameColumn('is_managers_service_request_resolved_notification_enabled', 'is_managers_service_request_closed_notification_enabled');
+            $table->renameColumn('is_auditors_service_request_resolved_email_enabled', 'is_auditors_service_request_closed_email_enabled');
+            $table->renameColumn('is_auditors_service_request_resolved_notification_enabled', 'is_auditors_service_request_closed_notification_enabled');
```

Comment:
In the future renames like this will need to be done more carefully with a transaction and feature flag update inline here and making sure the Model can handle both values.

But given that this is an unused feature. We are fine.